source/_integrations/osoenergy.markdown

```diff
@@ -23,10 +23,34 @@ This OSO Energy integration uses a subscription key, which a user can create for
 
 ## Platforms
```

Comment:
```suggestion
```
Not related to your PR, but we stopped referring to platforms in user documentation.